src/transpiling/jsx_precompile.rs

```diff
@@ -737,18 +882,14 @@ impl JsxPrecompile {
     strings: &mut Vec<String>,
     dynamic_exprs: &mut Vec<Expr>,
   ) {
-    for child in children.iter() {
+    let (normalized_children, _text_count, _serializable_count) =
+      merge_serializable_children(children);
+
+    for child in normalized_children {
       match child {
         // Case: <div>foo</div>
         JSXElementChild::JSXText(jsx_text) => {
-          let text = jsx_text_to_str(jsx_text);
-
-          // Text nodes which only contain whitespace can be ignored
-          if text.is_empty() {
-            continue;
-          }
-
-          let escaped_text = escape_html(text.as_ref());
+          let escaped_text = escape_html(jsx_text.value.as_ref());
           strings.last_mut().unwrap().push_str(escaped_text.as_str());
         }
         // Case: <div>{2 + 2}</div>
```

Comment:
Below here on line 897, I believe we can now remove the reference and therefore the clone of `expr`. Let me know if you'd like me to do a quick pass on this module.

Reply:
@dsherret Feel free to do a pass on this. Would appreciate it and would help me learn better as for me it's often easier to just see what it's supposed to look like and follow the train of thought from there 👍

Reply:
Sounds good. Doing it now!

Reply:
Should be good to go now.